qcodes/tests/test_parameter.py

```diff
@@ -1214,6 +1229,9 @@ def test_context(self):
             assert self.instrument.a.get() == 3
         assert self.instrument.a.get() == 2
 
+    def test_validated_param(self):
```

Comment:
adding precondition explicitly :)
```suggestion
    def test_validated_param(self):
        assert self.instrument.validated_param.get_latest() is None
```